src/main/java/mate/academy/dao/MovieSessionDao.java

```diff
@@ -0,0 +1,13 @@
+package mate.academy.dao;
+
+import java.time.LocalDate;
+import java.util.List;
+import mate.academy.model.MovieSession;
+
+public interface MovieSessionDao {
+    MovieSession add(MovieSession movieSession);
+
+    MovieSession get(Long id);
```

Comment:
Per the checklist, the `get()` method in DAO should return `Optional<MovieSession>` instead of `MovieSession` directly. This is to properly handle the case when a `MovieSession` with the given ID does not exist.